engineapi/instance_manager.go

```diff
@@ -16,11 +16,11 @@ import (
 )
 
 const (
-	CurrentInstanceManagerAPIVersion = 2
+	CurrentInstanceManagerAPIVersion = 3
 	MinInstanceManagerAPIVersion     = 1
 	UnknownInstanceManagerAPIVersion = 0
 
-	CurrentInstanceManagerProxyAPIVersion = 2
+	CurrentInstanceManagerProxyAPIVersion = 3
```

Comment:
If changing the existing proxy interface method like parameter change, need to update the min as well.